### ISSUE TITLE
Remove embargo from work accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#4003]
### Descriptive summary

QA Testing for Hyrax
Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 9b2360f
© 2019 Samvera-Hyrax
Nurax Version
ENV: production
SHA: aa07ac6ed440e9f700cce91fc602c115482e6bed
Branch: 2019-09-16-10-00
Updated: 16 Sep 2019 15:00:28

In testing for the Hyrax 3.0 release, the following points are actions that were discovered during a FireFox tab test and axe accessibility audit. Note, only certain issues are being reported at this time

**ARIA role must be appropriate for the element**

![Screen Shot 2019-09-16 at 12 27 22 PM](https://user-images.githubusercontent.com/9621070/64987102-5cbe5580-d87d-11e9-8c76-c459140ce6e3.png)

**Ensures landmarks are unique**

![Screen Shot 2019-09-16 at 12 31 00 PM](https://user-images.githubusercontent.com/9621070/64987375-e2da9c00-d87d-11e9-97a8-2014e9af6091.png)

### Steps to reproduce the behavior

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open manage embargo page from a work with an embargo. Need to be in edit mode. 
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Tab Test 

In Firefox and Safari, on the Manage Embargoes page, unable to tab through all the links and buttons. 

Chrome works fine. 

![Screen Recording 2019-09-16 at 12 29 PM](https://user-images.githubusercontent.com/9621070/64987241-a7d86880-d87d-11e9-95f1-b2fcdc1f7b25.gif)
